src/Reflection/StringCast/ReflectionClassConstantStringCast.php

```diff
@@ -16,13 +19,14 @@ final class ReflectionClassConstantStringCast
     public static function toString(ReflectionClassConstant $constantReflection) : string
     {
         $value = $constantReflection->getValue();
+        assert($value === null || is_scalar($value));
```

Comment:
A constant can technically return an array. This makes sure that it doesn't happen